config/confighttp/README.md

```diff
@@ -71,6 +71,9 @@ will not be enabled.
 [cors-headers]: https://developer.mozilla.org/en-US/docs/Glossary/CORS-safelisted_request_header
 [cors-cache]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age
 [origin]: https://developer.mozilla.org/en-US/docs/Glossary/Origin
+[attribute-processor]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/attributesprocessor/README.md
+
+you can enable [`attribute processor`][attribute-processor] to append `X-Forwarded-For` value to span's attribute using custom key.
```

Comment:
Any HTTP header can be added, not only this one.

Reply:
yes.  I'll fix it